cpr/session.cpp

```diff
@@ -180,6 +180,15 @@ void Session::prepareCommonShared() {
             curl_easy_setopt(curl_->handle, CURLOPT_PROXYPASSWORD, proxyAuth_.GetPassword(protocol));
         }
     }
+    // handle NO_PROXY override passed through Proxies object
+    // Example: Proxies{"no_proxy": ""} will override environment variable definition with an empty list
+    const std::string no_proxy[] = {"no_proxy", "NO_PROXY"};
+    for (const auto& item : no_proxy) {
```

Comment:
How about `std::any_of`?

Reply:
Does not seem that any_of would add readability.